frontend/src/pages/auth/Login.jsx

```diff
@@ -11,12 +11,22 @@ import { isBackDropLoaderDisplayed } from "src/redux/boolean/booleanSlice";
 import { useBackDropLoaderContext } from "src/context/BackDropLoaderContext";
 import { CommonLoaderWithBackDrop } from "src/common/loader/CommonLoader";
 import logoImage from "src/assets/icons/Logo.png";
+import InputAdornment from "@mui/material/InputAdornment";
+import Visibility from "@mui/icons-material/Visibility";
+import VisibilityOffIcon from "@mui/icons-material/VisibilityOff";
+import { useState } from "react";
+
 
 const Login = () => {
   const { handleSubmit, initialValues, isLoading, setFormValues } = useLogin();
 
   const dispatch = useDispatch();
 
+  const [toggle, setToggle] = useState(false);
+  const handleToggle = () => {
+    setToggle(!toggle);
+  }
+
```

Comment:
@vrun1208 please move this logic into uselogin hook. we try to don't write logic inside the component. I'm trying to maintain clean code.

Reply:
Sure! working on it. And good @NidhiSharma63! these are best practices for making clean code.

Reply:
@vrun1208 yes!